srcpkgs/faust/template

```diff
@@ -0,0 +1,52 @@
+# Template file for 'faust'
+pkgname=faust
+version=2.20.2
+revision=1
+build_style=gnu-makefile
+hostmakedepends="cmake pkg-config llvm"
+makedepends="libmicrohttpd-devel llvm-libunwind-devel"
+short_desc="Functional Programming Language for Real Time Signal Processing"
+maintainer="mag <mag-one@autistici.org>"
+license="GPL-2.0-or-later"
+homepage="http://www.faust.grame.fr/"
+distfiles=https://github.com/grame-cncm/faust/releases/download/${version}/${pkgname}-${version}.tar.gz
+checksum="c32588ee60d0f7ca25034d3d5436def6b35495f4d3beb3b02208d48684bcfbe9"
+nostrip_files="libsndfile.so "
+noshlibprovides="libsndfile.so"
+
+case $XBPS_TARGET_MACHINE in
+	*-musl) makedepends+=" libexecinfo-devel";;
+esac
+
+# INFO:
+#
+# - Faust puts its own libraries in /usr/share/faust
+# - In the same dir there's the Android app template
+#   used by the 'faust2android' script.
+# - The Android app template contains 'libsndfile.so'
+#   for Android that causes the installation to fail:
+#
+# => faust-2.15.11_1: running post-install hook: 11-pkglint-elf-in-usrshare ...
+# => ERROR: faust-2.15.11_1: ELF files found in /usr/share:
+# => ERROR:    /usr/share/faust/android/app/lib/libsndfile/lib/armeabi-v7a/libsndfile.so
+# => ERROR:    /usr/share/faust/android/app/lib/libsndfile/lib/arm64-v8a/libsndfile.so
+# => ERROR: faust-2.15.11_1: cannot continue with installation!
+#
+
+# FIRST (LAZY) WORKAROUND:
+#
+#post_install() {
+# rm -rf $PKGDESTDIR/usr/share/faust/android/
```

Comment:
I would just remove it instead of skiprdeps or even patch out the android dir in the build script, so it doesn’t even build android stuff at all.

Reply:
that was my first solution, but this way the 'faust2android' script will not work

Reply:
skiprdeps is the correct solution if the library is necessary, as the manual states:

> `skiprdeps` White space separated list of filenames specified by their absolute path in the $DESTDIR which will not be scanned for runtime dependencies. This may be useful to skip files which are not meant to be run or loaded on the host but are to be sent to some target device or emulation..